plugins/devtools-backend/README.md

```diff
@@ -44,7 +47,7 @@ Here's how to get the DevTools Backend up and running:
      const devToolsEnv = useHotMemoize(module, () => createEnv('devtools'));
      // ...
      // Insert this line under the other lines that add their routers to apiRouter in the same way
-     apiRouter.use('/devtools', await devTools(devToolsEnv));
+     apiRouter.use('/devtools', await devTools(devToolsEnv, [searchEnv.scheduler]));
```

Comment:
Would it make sense to call this out a bit in its own section as to how to use it? Here you have just the `searchEnv.scheduler` but for the example app you have `catalogEnv.scheduler` as well. Feel like I'm not catching on, maybe other will have the same problem but may not

Reply:
Maybe this could be separated in the instructions.